<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="252d427b-24d3-4782-8aba-0a89fb6c4822" name="fora@DELL 2015-08-18 02:25:53" runUser="dell\fora" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="ae7f9b34-cdf6-4f5e-886b-aeb8da462ab8">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="fora_DELL 2015-08-18 02_25_53">
      <DeploymentItem filename="B:\студия\Курсы C#\Task4\Data\Contacts\Contacts\Contacts\bin\Debug\Contacts.exe" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2015-08-18T02:25:53.6323311+06:00" queuing="2015-08-18T02:25:53.6624261+06:00" start="2015-08-18T02:25:53.6684299+06:00" finish="2015-08-18T02:25:53.9436251+06:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="ChangeCardNameTest_CheckNameEmptyException" storage="b:\студия\курсы c#\task4\data\contacts\contacts\unittest\bin\debug\unittest.dll" id="bb9dd28b-c4c3-c7d7-57b9-f57164bc92e2">
      <Execution id="92865c3f-544f-402f-b143-d6d19bec2547" />
      <TestMethod codeBase="B:/студия/Курсы C#/Task4/Data/Contacts/Contacts/Unittest/bin/Debug/Unittest.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="Unittest.CardTest, Unittest, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="ChangeCardNameTest_CheckNameEmptyException" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="bb9dd28b-c4c3-c7d7-57b9-f57164bc92e2" executionId="92865c3f-544f-402f-b143-d6d19bec2547" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="92865c3f-544f-402f-b143-d6d19bec2547" testId="bb9dd28b-c4c3-c7d7-57b9-f57164bc92e2" testName="ChangeCardNameTest_CheckNameEmptyException" computerName="DELL" duration="00:00:00.0427956" startTime="2015-08-18T02:25:53.6704344+06:00" endTime="2015-08-18T02:25:53.9296174+06:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="92865c3f-544f-402f-b143-d6d19bec2547">
      <Output>
        <ErrorInfo>
          <Message>Test method Unittest.CardTest.ChangeCardNameTest_CheckNameEmptyException threw exception System.ArgumentException, but exception System.ArgumentNullException was expected. Exception message: System.ArgumentException: Имя контакта не может быть пустым</Message>
          <StackTrace>    в Contacts.Card.ChangeCardName(String name) в B:\студия\Курсы C#\Task4\Data\Contacts\Contacts\Contacts\Card.cs:строка 55
   в Unittest.CardTest.ChangeCardNameTest_CheckNameEmptyException() в B:\студия\Курсы C#\Task4\Data\Contacts\Contacts\Unittest\CardTest.cs:строка 150
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>